salt/pillar/elife-dashboard.sls

```diff
@@ -31,3 +31,22 @@ elife_article_scheduler:
 
     web:
         port: 8080
+
+elife_dashboard_2:
```

Comment:
Not sure you'd need anything here apart from `name` and `web.port`?

The direction would be that we eventually transition to a single implementation, so the pillars get transferred to `elife_dashboard`. Pillars are sometimes difficult to modify because of their sourcing from `builder-private`.

Note: for this PR to be able to build, the pillars that are actually added here need to be put into `builder-private/pillar/elife-dashboard.sls` too (they should be ignored by the current formula).

Reply:
Have reduced it to `name`, `web.port` and `secret_key`.

Reply:
Have created a [PR here](https://github.com/elifesciences/builder-private/pull/138) for builder private